examples/tutorials/Understanding_WDGraphs_for_polymers.ipynb

```diff
@@ -0,0 +1,1289 @@
+{
```

Comment:
<p>reduce the image size</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
done